src/main/webapp/WEB-INF/views/courseForm.jsp

```diff
@@ -0,0 +1,28 @@
+<%@ taglib uri="http://www.springframework.org/tags/form" prefix="form"%>    
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%> 
+  
+ <h1>Add New Course</h1>  
+<form:form method="post" action="save">    
+ <table >    
```

Comment:
HTML: While the tutorial used a table, generally tables shouldn't be used for presentation/alignment of non-tabular info. This can be achieved with CSS.